### DISCUSSION TITLE
Search no longer jumping to result on Windows Desktop app

### DISCUSSION THREAD

**Discussion by snakespan** [standardnotes/forum#2579] [General]
**Describe the bug**
On desktop, you should be able to search in the search bar, and then once clicking on the note, it would 'jump' to the highlighted result in the note. This no longer happens. When I search on desktop Windows app, I get the highlighted result but I need to scroll to the result. I believe this behaviour regressed with one of the latest versions.

Infrequently, I'm also seeing results for keywords that are not actually found. If I copy my note into a text editor like Sublime, then search for the same thing, I see no results sometimes, even though Standard Notes has bizarrely marked a search as having results.

**To Reproduce**
1. Create a big note - enough so that you get BTL scroll
2. Search for a result
3. Observe that search doesn't jump to the result when clicking on the relevant note

**Expected behavior**
Search should jump to the highlighted result. Should only find results when the text matches.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - App Version 3.14.1

**Comment by maiertech** [comment]
I can confirm this behavior for macOS 12.3.1 with desktop app 3.14.1.

This problem points at a bigger pain point: finding information in my notes via search. I am aware that end-to-end encryption makes this more difficult, since you have to index locally and maybe sync the search index or create an index on each device.

But search really is my biggest pain point with Standard Notes.
